.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+Please give a clear and concise description of what the bug is.
+
+**To Reproduce**
+If you can describe the steps needed to reproduce here that will help us squash the bug faster.  If you can reproduce the bug using simple code, please put that code here.  Github supports [markdown](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax), you can surround code snippets by three backticks \`\`\` to show the code elegantly in the bug.
+
+**Cirq version**
+Please describe what version of Cirq you are using.  You can learn this by running `python -c 'import cirq; print(cirq.__version__)'`.
```

Comment:
Suddenly we're in the command line context; could be confusing. Just `print(cirq.__version__)` might be better.

Reply:
also, do we know a good command that would result in us knowing: 
- OS 
- python version 
- versions for cirq dependencies in the current Python PATH? 

Or should we write one?